src/main/java/com/xjeffrose/xio/http/internal/Http2HeadersWrapper.java

```diff
@@ -578,10 +582,15 @@ public int size() {
     return delegate.size();
   }
 
-  // TODO(CK): Remove this in favor of using the iterator constructor?
-
   @Override
-  public HttpHeaders http1Headers() {
-    return null;
+  public HttpHeaders http1Headers(boolean isTrailer, boolean isRequest) {
+    try {
+      HttpHeaders headers = new DefaultHttpHeaders();
+      HttpConversionUtil.addHttp2ToHttpHeaders(
+          -1, delegate, headers, HttpVersion.HTTP_1_1, isTrailer, isRequest);
+      return headers;
+    } catch (Http2Exception e) {
+      throw new RuntimeException(e);
```

Comment:
Document, maybe by adding to the interface documentation?

This should probably be a custom exception, also...

Reply:
Documentation, yes. Custom Exception? I dunno. I'm just trying to quash the checked exception here. We should maybe establish a policy around checked exceptions.

Reply:
Checked exceptions kind of suck, but they're deeply ingrained in Java libraries and general standards. It's accepted style, and we probably shouldn't fight that without good reason.

Regardless of whether it's checked or not, it should probably be a custom type.